tools/node.js

```diff
@@ -73,11 +73,13 @@ foam.CLASS({
     function execute() {
       var modelArgs = this.modelArgs;
       var X = this.__subContext__;
-      return X.arequire(this.modelId).then(function(model) {
-        return model.create(modelArgs, X).execute();
-      }).catch(function(err) {
-        console.log(err.stack);
-      });
+      var modelId = this.modelId;
+      return X.arequire(modelId)
+        .catch(console.log)
```

Comment:
I think so. I can run the following line from the browser:
new Promise(function() { throw 'hello' }).catch(console.log);
and it just logs 'hello'